### PR TITLE
SLING-11323: Sling update to 47

### PR DESCRIPTION
**Pull request by apelluru**

### CODE REVIEW

File: src/main/java/org/apache/sling/jcr/davex/impl/servlets/AuthHttpContext.java [hunk 1]
```diff
@@ -22,26 +22,20 @@
...
-    @Property(name = Constants.SERVICE_VENDOR, value = "The Apache Software Foundation"),
-    @Property(name = HttpWhiteboardConstants.HTTP_WHITEBOARD_CONTEXT_NAME, value = AuthHttpContext.HTTP_CONTEXT_NAME),
-    @Property(name = HttpWhiteboardConstants.HTTP_WHITEBOARD_CONTEXT_PATH, value = "/"),
-    @Property(name = Constants.SERVICE_RANKING, intValue = 5)
-})
+@Component(service = ServletContextHelper.class)
+@HttpWhiteboardContext(name = AuthHttpContext.HTTP_CONTEXT_NAME, path = "/")
+@ServiceDescription("Sling JcrRemoting Servlet")
+@ServiceVendor("The Apache Software Foundation")
+@ServiceRanking(4)
```
Comment (rombert) [inline src/main/java/org/apache/sling/jcr/davex/impl/servlets/AuthHttpContext.java]: The old service ranking was 5, why did you change it to 4?

**Review by rombert — CHANGES_REQUESTED** [review]

File: src/main/java/org/apache/sling/jcr/davex/impl/servlets/AuthHttpContext.java (hunk 1 above)
Comment (apelluru) [reply src/main/java/org/apache/sling/jcr/davex/impl/servlets/AuthHttpContext.java]: Sorry I was testing ranking orders, Thanks for highlighting it

**Review by rombert — APPROVED** [review]
Looks good, thanks @ashokmca07 !
